db/sql_implementation.go

```diff
@@ -307,6 +339,8 @@ const updateMetadataQuery = `UPDATE metadata SET
 
 func (db *DB) migrate() error {
 	_, err := db.sqldb.ExecContext(db.ctx, schema)
```

Comment:
Right now the errs for the first two `ExecContext` calls can get swallowed. can you just use a for loop for the three schemas?

Reply:
Yes, to be fixed (linters also warn for this) before moving from draft :smile: